wger/core/api/serializers.py

```diff
@@ -26,6 +28,18 @@
     WeightUnit)
 
 
+class UserRegistrationSerializer(serializers.ModelSerializer):
```

Comment:
we can have this being more general (`UserSerializer`). In case we need to use it to also list the users